src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -1613,6 +1628,15 @@ public void GetPartsOfBinaryExpression(SyntaxNode node, out SyntaxNode left, out
             right = binaryExpression.Right;
         }
 
+        public void GetPartsOfBinaryExpression(SyntaxNode node, out SyntaxNode left, out SyntaxNode right, out SyntaxToken token)
+        {
+            GetPartsOfBinaryExpression(node, out left, out right);
```

Comment:
i would do things the reverse way.  have the Simple version call into this version.